document/JS.bs

```diff
@@ -0,0 +1,935 @@
+<pre class='metadata'>
+Title: WebAssembly JS API
+H1: WebAssembly
+Shortname: wasmjs
+Group: WebAssembly CG
+Status: w3c/CG-DRAFT
+Level: 1
+ED: https://webassembly.github.io/spec/JS.html
+Editor: WebAssembly Community Group
+Repository: WebAssembly/spec
+Abstract: This document provides an explicit JavaScript API for interacting with WebAssembly.
+Markup Shorthands: css no, markdown yes
+Ignored Terms: h1, h2, h3, h4, h5, h6, xmp
+</pre>
+
+<pre class='biblio'>
+{
+  "ECMA-262": {
```

Comment:
This is not necessary. Just use [ECMASCRIPT] and it'll work.

Reply:
More cargo-culting; fixed.